src/components/Tooltip/Tooltip.tsx

```diff
@@ -19,7 +19,7 @@ const Tooltip = ({
   events = ['hover'],
   positionStrategy = 'absolute',
   middlewares,
-  wrapper: WrapperElement = 'div',
```

Comment:
default value removed because there's no way to use Tooltip component directly without Controller and Controller always passes a `wrapper` to Tooltip